src/VisualStudio/Core/Impl/ProjectSystem/CPS/CPSProject_IWorkspaceProjectContext.cs

```diff
@@ -199,7 +199,7 @@ public void SetProperty(string name, string? value)
             }
             else if (name == BuildPropertyNames.CompilerGeneratedFilesOutputPath)
             {
-                _projectSystemProject.GeneratedFilesOutputDirectory = string.IsNullOrWhiteSpace(value) ? null : value;
+                _projectSystemProject.GeneratedFilesOutputDirectory = PathUtilities.IsAbsolute(value) ? value : null;
```

Comment:
If a user sets `EmitCompilerGeneratedFiles` to `true` in their project, then `CompilerGeneratedFilesOutputPath` evaluates to a relative path by default. With this change, the value will be ignored, which seems like an issue.

I don't see any docs for `CompilerGeneratedFilesOutputPath`, but all the examples I see online use a relative value:.

- https://github.com/dotnet/roslyn/issues/49981
- https://stackoverflow.com/a/68320185/24874
- https://www.reddit.com/r/dotnet/comments/mrgx3u/how_to_put_source_generator_code_into_source/